src/Form/Form.js

```diff
@@ -233,7 +236,7 @@ class Form {
         if (input.nodeName !== 'SELECT') {
             const events = ['pointerdown']
             if (!isMobileApp) events.push('focus')
-            events.forEach((ev) => this.addListener(input, ev, handler, true))
```

Comment:
Any reason to remove the capture flag? These usually ensure our listeners run before the page's own listeners.

Reply:
yeah, because `this.addListener` doesn't accept a 4th param

https://github.com/duckduckgo/duckduckgo-autofill/blob/91229cef823fcd75b1fcc03eae51414ae9a385f9/src/Form/Form.js#L186-L189

Reply:
Ah, yeah, of course. Brainfart.